in/index.html.mako

```diff
@@ -369,6 +376,17 @@
               <span sort="999999">unavailable</span>
             % endif
           </td>
+          <td class="emr">
```

Comment:
The `data-pricing` attribute should be provided here so that the costs for other regions can be viewed like in the other cost columns.

Reply:
Ack